.github/workflows/release.yml

```diff
@@ -0,0 +1,19 @@
+---
+name: release
+
+on:
+  push:
+    tags:
+      - v*.*.*
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Release Gem
+        uses: cadwallion/publish-rubygems-action@master
```

Comment:
Please do not link to master. This make the action unpredictable, we may end up running uncontrolled executions.

We should AT LEAST aim to a particular vetted version, or tag. Even https://github.com/marketplace/actions/publish-to-rubygems suggests a specific version.

```suggestion
       # v1.0.0
        uses: cadwallion/publish-rubygems-action@9493635
```

I'd point to a specific release in this way it's more hard to retag it.